src/activities/Elsa.Activities.Http/Activities/SendHttpRequest/SendHttpRequest.cs

```diff
@@ -32,13 +32,15 @@ public class SendHttpRequest : Activity, IActivityPropertyOptionsProvider
     {
         private readonly HttpClient _httpClient;
         private readonly IEnumerable<IHttpResponseContentReader> _parsers;
+        private readonly List<ISendHttpRequestAuthorizationHeaderHandler> _authorizationHeaderHandlers;
 
         public SendHttpRequest(
             IHttpClientFactory httpClientFactory,
-            IEnumerable<IHttpResponseContentReader> parsers)
+            IEnumerable<IHttpResponseContentReader> parsers, IEnumerable<ISendHttpRequestAuthorizationHeaderHandler> authorizationHeaderHandlers)
```

Comment:
Then it will fall into the same trap as current credential manager implementation - you will need to publish every workflow again whenever credential is changing.